lib/src/track/options.dart

```diff
@@ -102,15 +102,16 @@ class ScreenShareCaptureOptions extends VideoCaptureOptions {
   final bool captureScreenAudio;
 
   /// for browser only, if true, will capture current tab.
-  final bool preferCurrentTab;
+  @Deprecated('Deprecated parameter no longer works in chrome m115+')
```

Comment:
I don't think this is deprecated.. what do you mean it no longer works?

Reply:
I think the problem is that we've defaulted it to true.. the default should be false.

Reply:
ah, okay.